src/features/Status/StatusPage.tsx

```diff
@@ -63,25 +63,38 @@ class StatusPage extends React.Component<IStatusPageProps, {}> {
                     <Button type="button">View/Edit Application</Button>
                   </LinkDuo>
                 </Flex>
+              ) : Date.now() < 1578070799000 ? (
```

Comment:
same for here, can you comment the date this number means and where you generated? might actually be good to make this number a constant in the constants file and import it